lib/theme-options/site-options.php

```diff
@@ -28,21 +28,10 @@
       'options' => get_post_objects(
         array(
           'post_type' => 'evento',
+          'numberposts' => -1,
```

Comment:
where does this variable come from? It isn't normal: https://codex.wordpress.org/Template_Tags/get_posts

Reply:
Should it be `posts_per_page` or am I missing something

Reply:
`Note: 'numberposts' and 'posts_per_page' can be used interchangeably.`

Reply:
it is normal. for get_posts

Reply:
Surely `posts_per_page` is correct if that is how it is always documented? I guess either a forthcoming change to the codebase or a hangover from a previous approach.

Reply:
Interesting: https://stackoverflow.com/questions/3334711/showposts-vs-posts-per-page-showposts-works-perfect-but-deprecated Seems to suggest that if you are doing `get_posts` which is not a WP_Query and not paginated then `numberposts` is better semantically even tho they are interchangeable. Seems like a kinda ugly WP quirk really but still this code is correct.